tap_closeio/streams.py

```diff
@@ -42,10 +42,15 @@ def bookmark(tap_stream_id):
 
 
 def new_max_bookmark(max_bookmark, records, key):
+    now = datetime.utcnow()
```

Comment:
Could this value be pegged at the start of the job? There might be a race condition here where `utcnow()` advances beyond a certain point during the course of extraction. Probably not going to cause a data discrep since closeio isn't respecting our `lt`, but if they start to, it might.